website/content/docs/troubleshooting.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Troubleshooting Guide
```

Comment:
Expert might be a bit strong a word. :)

The other one is called Troubleshooting Guide in the left nav, but 'Topology Troubleshooting Guide' on the page.
http://twitter.github.io/heron/docs/developers/troubleshooting/

This one also seems like a Topology Troubleshooting Guide IMO, but if we want to have two guides, maybe this is Topology Deployment Troubleshooting Guide? @lucperkins do you have a suggestion here?